test/suites/apparmor.sh

```diff
@@ -0,0 +1,113 @@
+test_apparmor() {
+  if [ ! -e /sys/module/apparmor/ ]; then
+    export TEST_UNMET_REQUIREMENT="missing AppArmor kernel support"
+    return
+  fi
+
+  ensure_import_testimage
+
+  echo "Create and start a test container"
+  lxc launch testimage c1
+
+  echo "==> Test /sys/kernel/* access"
+
+  echo "Verify access to /sys/kernel/config denied"
+  ! lxc exec c1 -- ls /sys/kernel/config/ || false
+
+  echo "Verify access to /sys/kernel/debug denied"
+  ! lxc exec c1 -- ls /sys/kernel/debug/ || false
+
+  echo "Verify access to /sys/kernel/tracing denied"
+  ! lxc exec c1 -- ls /sys/kernel/tracing/ || false
+
+  echo "Verify read access to /sys/kernel/security allowed"
+  lxc exec c1 -- ls /sys/kernel/security/
+
+  echo "Verify write access to /sys/kernel/security denied"
+  ! lxc exec c1 -- touch /sys/kernel/security/testfile || false
+
+  echo "Verify write access to /sys/kernel/security/apparmor allowed"
+  lxc exec c1 -- touch /sys/kernel/security/apparmor/.load
+
+  echo "==> Test /sys/firmware/efi/efivars/ access"
+
+  if lxc exec c1 -- test -d /sys/firmware/efi/efivars/; then
```

Comment:
Aleks would know for sure but I assumed this depended on the `efivars` existing in the host itself.